client/src/app/ui/modules/sorting/modules/sorting-list/components/sorting-list/sorting-list.component.html

```diff
@@ -9,8 +9,10 @@
             cdkDrag
             class="line margin-bottom-5"
             [ngClass]="isSelectedRow(i) ? 'backgroundColorSelected' : 'backgroundColorLight'"
+            [tabindex]="0"
             (cdkDragStarted)="dragStarted(i)"
             (click)="onItemClick($event, i)"
+            (keyup)="onItemClick($event, i)"
```

Comment:
I assume with this change it should be possible to change the order of the os-sorting-lists with just the keyboard
Please explain with what actions/keys it should be possible to change the order
I tried enter, space, arrow keys and ctrl but no item I focus moved in any direction

Furthermore, I was not able to focus the agenda-items and category-items at all

Reply:
No, this is to select multiple Items.